SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorSettings.cpp

```diff
@@ -464,8 +464,11 @@ bool USpatialGDKEditorSettings::IsDeploymentConfigurationValid() const
 		}
 	}
 
+	FString LaunchConfigPath = GetPrimaryLaunchConfigPath();
+	TUniquePtr<FArchive> ConfigFile(IFileManager::Get().CreateFileReader(*LaunchConfigPath));
+
 	TArray<FString> WorkersManuallyLaunched;
-	if (IsManualWorkerConnectionSet(GetPrimaryLaunchConfigPath(), WorkersManuallyLaunched))
+	if (ConfigFile != nullptr && IsManualWorkerConnectionSet(LaunchConfigPath, WorkersManuallyLaunched))
```

Comment:
Could we use `IFileManager::Get().FileExists` instead?

Reply:
Also, it seems like we're doing the same here inside `IsManualWorkerConnectionSet`. Perhaps we should just not error if the file isn't there.